README.md

```diff
@@ -92,7 +92,7 @@ The following can normally be used:
 
 ## FAQ
 #### Q: How to recover if Jenkins keeps failing during the login phase?
-A: You can disable the security from the config file (see https://www.jenkins.io/doc/book/system-administration/security/#disabling-security)
+A: You can disable the security from the config file (see [https://www.jenkins.io/doc/book/security/access-control/disable/](https://www.jenkins.io/doc/book/security/access-control/disable/))
```

Comment:
Could perhaps put <> around the URL instead of repeating it

```suggestion
A: You can disable the security from the config file (see <https://www.jenkins.io/doc/book/security/access-control/disable/>)
```

But I have not checked how that renders on GitHub and at plugins.jenkins.io.

Reply:
not sure :), this will work as is anyway